lib/gh-koenig/.gitignore

```diff
@@ -0,0 +1,55 @@
+b-cov
```

Comment:
For some reason it commited all of the ember stuff for me and I had to reverse my commit. Maybe it's not required.